### PR TITLE
Moved database functions to separate file

### PR DESCRIPTION
**Pull request by kaysabelle**

### CODE REVIEW

File: radlibDemoApp/www/index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
         <link rel="stylesheet" type="text/css" href="css/index.css" />
         <title>RadLib Beta Demo</title>
 		<script type="text/javascript" src="js/index.js"></script>
+		<script type="text/javascript" src="js/database.js"></script>
```
Comment (jeslin) [inline radlibDemoApp/www/index.html:10]: WOW NEAT

File: radlibDemoApp/www/js/database.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+	 tx.executeSql('SELECT * FROM LOGS', [], function (tx, results){
+	 var len = results.rows.length, i;
+
+	  for (i = 1; i < len; i++) {
+	   csvData += results.rows.item(i).id + "," + results.rows.item(i).log + "\n";
+	   }
+	 window.location='data:text/csv;charset=utf8,' + encodeURIComponent(csvData);
+	  });
+ });
+}
```
Comment (jeslin) [inline radlibDemoApp/www/js/database.js:132]: LOOKS GUD

Comment (vinchan) [inline radlibDemoApp/www/js/database.js:132]: //VINCENT WAS HERE and approves

**Comment by DanielUShu** [comment]
Awyis

File: radlibDemoApp/www/js/database.js (hunk 2 above)
Comment (kaysabelle) [inline radlibDemoApp/www/js/database.js:132]: Thx yall

**Pull request merged**